content/usgs-packages/sbtools_Get.Rmd

```diff
@@ -0,0 +1,179 @@
+
+```{r sbtools-login, eval=FALSE}
+authenticate_sb()
+```
+
+To double check that your authentication was successful, use the function `is_logged_in`. It will return a logical to let you know if you are logged in or not. No arguments are needed.
+
+```{r sbtools-verifylogin}
+is_logged_in()
+```
+
+Each user has a specific ScienceBase id associated with their account. The user ids can be used to inspect what top-level items saved under your account (discussed in next section). To determine your user id, use the function `user_id` in an authenticated session. No arguments are necessary.
+
+```{r sbtools-userid}
+user_id()
+```
+
+When you're done with your session, you can actively logout using the `session_logout`. No arguments are required. If you do not do this, you will be automatically logged out after a certain amount of time or when you close R.
+
+## Inspect and download items
+
+The first inspection step for ScienceBase items is to determine if the item even exists. To do this, use the function `identifier_exists`. The only required argument is `sb_id` which can be either a character string of the item id or an `sbitem`. It will return a logical to indicate if the item exists or not.
+
+```{r sbtools-identifierexists}
+identifier_exists("4f4e4acae4b07f02db67d22b")
+identifier_exists("thisisnotagoodid")
+```
+
+ScienceBase items can be described by alternative identifiers, e.g. digital object identifiers, IPDS codes, etc. They are defined on ScienceBase with a scheme, type, and key. For examples of identifiers, see the "Additional Information | Identifiers" section of [Differential Heating](https://www.sciencebase.gov/catalog/item/580587a2e4b0824b2d1c1f23). 
+
+You can use the function `item_exists` to check whether or not a scheme-type-key tuple already exists. The function has three required arguments - `scheme`, `type`, and `key`. Note that the table of alternative identifiers on ScienceBase is in a different order than this function accepts. On ScienceBase: type, scheme, key. For `item_exists`: scheme, type, key.
+
+```{r sbtools-itemexists}
+# test a made up tuple
+item_exists(scheme = "made", type = "this", key = "up")
+
+# test a tuple from the SB item "4f4e4acae4b07f02db67d22b"
+item_exists(scheme = "State Inventory", type = "UniqueKey", key = "P1281")
+
+# test the same scheme & type with a made up key
+item_exists(scheme = "State Inventory", type = "UniqueKey", key = "1234")
+```
+
+You can create sbitems from just the ScienceBase id. To do this use `as.sbitem`. *why you would use it*
```

Comment:
i think you could safely leave out `as.sbitem` entirely and just show `item_get` here instead. see comment on `item_get` below.